evidence/claravanstaden/0001-claravanstaden-2024-retention-rank-1.md

```diff
@@ -0,0 +1,91 @@
+# Evidence-0001: Retention at Rank 1
+
+|                 |                |
+| --------------- |----------------|
+| **Report Date** | 2024/09/11     |
+| **Submitted by**| claravanstaden |
+
+
+## **Member details**
+
+* Matrix username: @claravanstaden:matrix.org
+* Polkadot address: 12aoZXwbUzsv3z5HF5HCrtEwBJYCeKne6rYsxFEKDZ86Wdv8
+* Current rank: 1
+* Date of initial induction: 2024/06/17
+* Date of last report: 2024/06/17
```

Comment:
Date of last report should be: N/A

Reply:
Fixed in https://github.com/polkadot-fellows/Evaluations/pull/60/commits/363aa62cbf11ef85b800b656d8773aed277c6cb3. :)